src/openrct2/network/NetworkBase.cpp

```diff
@@ -715,11 +715,66 @@ const char* NetworkBase::FormatChat(NetworkPlayer* fromplayer, const char* text)
 {
     static std::string formatted;
     formatted.clear();
-    formatted += "{OUTLINE}";
+
     if (fromplayer != nullptr)
```

Comment:
Could you use variable names that are in `camelCase`?

- `fromPlayer`
- `groupName`
- `proportionalSize`
- `numCharacters`

Additionally, we use British English in our codebase, so please rename anything with `color` to `colour`,

Reply:
Cameled and coloured!